### ISSUE TITLE
fix naming conventions

### ISSUE THREAD

**Issue by nikolojedison** [DenfeldRobotics4009/2015_Lopez_Jr#6]
Need to tidy up the names for folders of commands. Talk w/ Aux about the rest.

**Comment by auxiliary-character** [comment]
Nuh uh.

**Comment by nikolojedison** [comment]
The opportunity cost for this should be considered - is it truly worth it to go back through the code and simplify the folder names so that the convention is satisfied, or should we focus on teaching the code and what conventions we should start with?

**Comment by auxiliary-character** [comment]
Yeah, I don't think it's that huge of a priority, but it's something to keep in mind and maybe clean up this summer. We certainly shouldn't do it before state.

**Comment by nikolojedison** [comment]
Alright. It could be a way for people other than you and I to learn the code - where everything is, what things each command does & what they depend on, etc.
